### ISSUE TITLE
Unknown archive format after first try

### ISSUE THREAD

**Issue by hanoii** [webrecorder/pywb#432]
I am very new to this project. I checked out master, run the python install command, launched the recorder as per the usage and accessing `https://www.facebook.com` immediately gives me:

```
Pywb Error
Internal Error: Unknown archive format, first line: ['4000']
Error Details:

500 Server Error
```

I tried google, which works, but once I search something, I get the same again.

Do I need to do something else for this to work?

**Comment by ikreymer** [comment]
Hi, sorry for the delayed response.

Could you provide more details on how you're running pywb when you get the error?

Assuming you've running as per the instructions in the docs:

```
Create a collection: wb-manager init my-web-archive (if you haven’t already created a web archive collection)
Run: wayback --record --live -a --auto-interval 10
Point your browser to http://localhost:8080/my-web-archive/record/<url>
```

Do you see the 500 error when loading `http://localhost:8080/my-web-archive/record/https://facebook.com/` ? 

You can also add `--debug` command to get additional error reporting in the log.

**Comment by hanoii** [comment]
I tried with `--debug` and yes, i am doing it per the instructions in the doc:

```
2019-01-09 22:35:12,727: [DEBUG]: Starting new HTTP connection (1): localhost:50271
2019-01-09 22:35:12,736: [DEBUG]: Starting new HTTP connection (1): localhost:50270
2019-01-09 22:35:12,742: [DEBUG]: Starting new HTTPS connection (1): www.facebook.com:443
2019-01-09 22:35:13,142: [DEBUG]: https://www.facebook.com:443 "GET / HTTP/1.1" 200 None
2019-01-09 22:35:13,154: [DEBUG]: http://localhost:50270 "POST /live/resource/postreq?param.recorder.coll=test&url=https%3A%2F%2Fwww.facebook.com%2F&matchType=exact&closest=now HTTP/1.1" 200 None
2019-01-09 22:35:13,157: [DEBUG]: http://localhost:50271 "POST /live/resource/postreq?param.recorder.coll=test&url=https%3A%2F%2Fwww.facebook.com%2F&matchType=exact&closest=now HTTP/1.1" 200 None
Traceback (most recent call last):
  File "/Users/ariel/Sites/brickfactory/pywb/pywb/apps/frontendapp.py", line 493, in handle_request
    response = endpoint(environ, **args)
  File "/Users/ariel/Sites/brickfactory/pywb/pywb/apps/frontendapp.py", line 338, in serve_record
    return self.serve_content(environ, coll, url, record=True)
  File "/Users/ariel/Sites/brickfactory/pywb/pywb/apps/frontendapp.py", line 375, in serve_content
    response = self.rewriterapp.render_content(wb_url_str, metadata, environ)
  File "/Users/ariel/Sites/brickfactory/pywb/pywb/apps/rewriterapp.py", line 322, in render_content
    ensure_http_headers=True)
  File "/usr/local/lib/python2.7/site-packages/warcio-1.6.3-py2.7.egg/warcio/recordloader.py", line 88, in parse_record_stream
    known_format))
  File "/usr/local/lib/python2.7/site-packages/warcio-1.6.3-py2.7.egg/warcio/recordloader.py", line 215, in _detect_type_load_headers
    raise ArchiveLoadFailed(msg + str(se.statusline))
ArchiveLoadFailed: Unknown archive format, first line: ['4000']
127.0.0.1 - - [2019-01-09 22:35:13] "GET /test/record/mp_/https://www.facebook.com/ HTTP/1.1" 404 259 0.446179
127.0.0.1 - - [2019-01-09 22:35:13] "POST /live/resource/postreq?param.recorder.coll=test&url=https%3A%2F%2Fwww.facebook.com%2F&matchType=exact&closest=now HTTP/1.1" 200 117293 0.447143
127.0.0.1 - - [2019-01-09 22:35:13] "POST /live/resource/postreq?param.recorder.coll=test&url=https%3A%2F%2Fwww.facebook.com%2F&matchType=exact&closest=now HTTP/1.1" 200 16937 0.455774
127.0.0.1 - - [2019-01-09 22:35:13] "GET /favicon.ico HTTP/1.1" 307 163 0.000250
127.0.0.1 - - [2019-01-09 22:35:13] "GET /test/record/https://www.facebook.com/favicon.ico HTTP/1.1" 200 1283 0.000936
2019-01-09 22:35:14,766: [INFO]: Checking Collection: test
2019-01-09 22:35:14,767: [INFO]: Auto-Indexing... ['/Users/ariel/Sites/brickfactory/pywb/collections/test/archive/rec-20190110013513189322-nashville.warc.gz']
2019-01-09 22:35:14,805: [INFO]: ...Done
2019-01-09 22:35:24,807: [INFO]: Checking Collection: test
2019-01-09 22:35:34,809: [INFO]: Checking Collection: test
2019-01-09 22:35:44,811: [INFO]: Checking Collection: test
```

I called the collection test btw.

**Comment by hanoii** [comment]
I tried webrecorder as well, it worked slightly better, but at some point, facebook stops responding but i want to check this one first

**Comment by ikreymer** [comment]
Thanks for the update, I can repro the issue with python 2.7. I will take a look.
If you use install pywb with Python 3.5+, this issue does not appear.

**Comment by ikreymer** [comment]
Should now be fixed on develop, should work with python 2.7+, thanks for catching this!
